scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta

```diff
@@ -12,8 +12,11 @@ public class <%= it.className %> implements ModInitializer {
 	// This logger is used to write text to the console and the log file.
 	// It is considered best practice to use your mod id as the logger's name.
 	// That way, it's clear which mod wrote info, warnings, and errors.
-<% if (it.slf4j) { %>    public static final Logger LOGGER = LoggerFactory.getLogger("<%= it.modid %>");
-<% } else { %>    public static final Logger LOGGER = LogManager.getLogger("<%= it.modid %>");<% } %>
+        public static final String MOD_ID = "<%= it.modid %>";
```

Comment:
There should be a single tab, same as the comments above. I would also expect this line to be above the comments.